runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ResourceScriptResolver.java

```diff
@@ -40,12 +45,26 @@
         }
     }
 
+    private static String loadScriptFromResourceAsString(String resourceName) {
+        try (InputStream stream = ResourceScriptResolver.class.getClassLoader().getResourceAsStream(resourceName)) {
+            InputStream safeStream = requireNonNull(stream, "Could not locate resource '" + resourceName + "'");
+            // Must make sure that the eval.py does not have " character in its source.
+            return join(readLines(safeStream, UTF_8), "\n");
```

Comment:
Since we already have the eval script in bytes, stored in the class, why not return the string representation of that byte array? `new String(evalBytes, UTF-8)` ?

Reply:
I need to insert _\n_ character after each line and also making the fields dependent of each other seems a code smell to me.